mod-pubsub-server/src/main/java/org/folio/services/util/MessagingModulesUtil.java

```diff
@@ -0,0 +1,47 @@
+package org.folio.services.util;
+
+import org.folio.rest.jaxrs.model.MessagingModule;
+import org.folio.rest.util.MessagingModuleFilter;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.collections4.CollectionUtils.isNotEmpty;
+
+public final class MessagingModulesUtil {
+
+  private MessagingModulesUtil() {
+  }
+
+  public static List<MessagingModule> filter(List<MessagingModule> modules, MessagingModuleFilter filter) {
+    if (isNotEmpty(modules)) {
+      return modules.stream()
+        .filter(messagingModule -> satisfiesCondition(messagingModule, filter))
+        .collect(Collectors.toList());
+    }
+    return modules;
+  }
+
+  public static boolean satisfiesCondition(MessagingModule module, MessagingModuleFilter filter) {
+    boolean result = true;
+    if (filter.getEventType() != null) {
```

Comment:
I just wonder - what if "filter" will be empty? Then it will be better to add an extra check.